crdt.go

```diff
@@ -138,6 +163,19 @@ type Datastore struct {
 	curDelta    *pb.Delta // current, unpublished delta
 
 	wg sync.WaitGroup
+
+	jobQueue chan *dagJob
```

Comment:
hmm so originally there was only jobQueue, but it was impossible to ensure that this channel would be closed properly because there were multiple senders so I had to tunnel everything first through a single sender using the newJobs. Technically the job queue is the job queue. Perhaps the one we need to change is `newJobs`. I've renamed it to `sendJobs` which is processed by the `sendJobWorker` for the moment.